remoting/subsystem/src/main/java/org/jboss/as/remoting/ConnectorResource.java

```diff
@@ -72,6 +74,16 @@
             .setNullSignificant(true)
             .build();
 
+    static final SimpleAttributeDefinition SASL_AUTHENTICATION_FACTORY = new SimpleAttributeDefinitionBuilder(ConnectorCommon.SASL_AUTHENTICATION_FACTORY)
+            .setCapabilityReference(SASL_AUTHENTICATION_FACTORY_CAPABILITY, CONNECTOR_CAPABILITY)
```

Comment:
Downstream on the plan we have a separate task to go through and add the access constraints.  This is slightly different to what we had before so I would like to go through it in one go with Brian.